niceday-api/package.json

```diff
@@ -12,7 +12,7 @@
     "REST",
     "Niceday"
   ],
-  "license": "Apache 2.0",
+  "license": "Apache-2.0",
```

Comment:
Apparently this should be a valid SPDX identifier https://spdx.org/licenses/ otherwise npm gives you a warning.